rpcs3/Emu/Cell/SPUASMJITRecompiler.cpp

```diff
@@ -1043,10 +1043,24 @@ void spu_recompiler::IRET(spu_opcode_t op)
 
 void spu_recompiler::BISLED(spu_opcode_t op)
 {
-	fmt::throw_exception("Unimplemented instruction" HERE);
+	c->mov(asmjit::X86Gp t, SPU_OFF_32(gpr, op.ra, &v128::_u32, 3));
+	c->and_(t, 0x3fffc);
+	
+	const XmmLink& vr = XmmAlloc();
+	c->movdqa(vr, XmmConst(_mm_set_epi32(spu_branch_target(m_pos + 4), 0, 0, 0)));
+	c->movdqa(SPU_OFF_128(gpr, op.rt), vr);
+	//todo:
+	//if (/*external event*/){
+	//if (op.d || op.e) c->or_(*addr, op.e << 26 | op.d << 27); // interrupt flags stored to PC
+	// c->mov(SPU_OFF_32(pc), t);}
+	// else
+->> c->mov(SPU_OFF_32(pc), SPU_OFF_32(vr));
+	c->unuse(vr);
+	fmt::TODO!("SPU external event check" HERE);
```

Comment:
This is still not correct. HERE means nothing now. You should write something like this:
if (external event){
 LOG_ERROR(SPU, "BISLED External event");
}

Reply:
@woj1993  but will it go further in the code after this line?

Reply:
After if yes. After Log_ERROR yes. If you want it to stop Exception is best metod but then you must have if becouse it will stop/crash evrytime when this function will be called.